src/Makefile.am

```diff
@@ -691,28 +720,18 @@ libzcash_a_CXXFLAGS = $(AM_CXXFLAGS) $(PIE_FLAGS)
 if BUILD_BITCOIN_LIBS
 include_HEADERS = script/zcash_script.h
 libzcash_script_la_SOURCES = \
-  crypto/hmac_sha512.cpp \
-  crypto/ripemd160.cpp \
-  crypto/sha1.cpp \
-  crypto/sha256.cpp \
-  crypto/sha512.cpp \
-  hash.cpp \
-  primitives/transaction.cpp \
-  primitives/tx_version_info.cpp \
-  pubkey.cpp \
-  script/zcash_script.cpp \
-  script/interpreter.cpp \
-  script/script.cpp \
-  uint256.cpp \
-  util/strencodings.cpp \
-  zip317.cpp
+  $(crypto_libbitcoin_crypto_a_SOURCES) \
+  $(libbitcoin_script_a_SOURCES) \
+  $(libcxxbridge_a_SOURCES) \
+  support/cleanse.cpp \
```

Comment:
The only reason this is needed here, is because if we try to instead move it into `libbitcoin_script` we run into linker errors in `zcash-cli` that only exist because the Rust binaries are a monolith. If/when we get everything `cxx`-ified, we can probably break out those specific dependencies into a separate bridge.